amplitude/.circleci/config.yml

```diff
@@ -0,0 +1,134 @@
+version: 2
```

Comment:
Let's add that to the root directory, under `.circleci/`.

Reply:
Okay 👍